etc/profile-m-z/parsecd.profile

```diff
@@ -0,0 +1,22 @@
+include parsecd.local
+include globals.local
+
+whitelist ${HOME}/.parsec
```

Comment:
To protect a program's configuration path, it's necessary to (1) blacklist it in `disable-programs.inc`, (2) noblacklist it in the program's dedicated profile and (3) then proceed with any whitelisting. Again, use the template for guidance here.